src/app/components/GearSelector/GearSelector.tsx

```diff
@@ -72,6 +72,11 @@ export default function GearSelector({
           />
         </label>
       </form>
+
+      <div className={style.optionsWrapper}>
+        <Typography size="s" color="light" children="Select options" />
```

Comment:
`<Typography size="s" color="light">Select options</Typography>` would've been better

Reply:
Resolved in [36c05f7](https://github.com/Seborider/plugs./pull/28/commits/36c05f76b4e1c15d2b8bb08aee8bfb65aa37d3cf)